README.md

```diff
@@ -165,16 +167,17 @@ procedure:
 
 - Do a quick test of https://glean-dictionary-dev.netlify.app to make sure it's
   working as expected.
-- Create a new release, typically off of the `main` branch (use the
-  [auto-generated release notes](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes),
-  omitting dependency updates).
-- From a local checkout, update the `production` branch to be in sync with the
-  tag you just created, then push to the production branch. After the
-  integration tests pass, dictionary.telemetry.mozilla.org should be
-  automatically updated to the latest version.
-
-A version of the Glean Dictionary running the development branch (`main`) is
-accessible at https://glean-dictionary-dev.netlify.app/
+- Create a new release off of the `main` branch:
+  - use the [auto-generated release notes](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes),
+  omitting dependency updates, unless it's `glean.js`;
+  - use the format `vX.Y.Z` for the tag, where `X.Y.Z` is the new version number.
+- From a local checkout (assuming `origin` is the name of the remote):
+  - fetch the newly created tags, `git fetch --tags origin`;
+  - switch to the `production`, `git checkout production`;
+  - make it in sync with the tag you just created, `git merge tags/vX.Y.Z` (where `X.Y.Z` is the new version number).
+  - push to the production branch, `git push origin production`.
+- Wait for the integration tests to pass by monitoring [CircleCi](https://app.circleci.com/pipelines/github/mozilla/glean-dictionary?branch=production).
+- Check that dictionary.telemetry.mozilla.org is automatically updated to the latest version.
```

Comment:
```suggestion
- Check that https://dictionary.telemetry.mozilla.org is automatically updated to the latest version.
```

Gets you the auto-linking, so one can click it.

Reply:
Maybe call out that it lists the deployed commit on the bottom? "Built from revision [a17b6f8cb9](https://github.com/mozilla/glean-dictionary/tree/a17b6f8cb9e41caa6e9f8f97149aa443c6ca7d4e)."

Reply:
Good idea!